packages/packages.csv

```diff
@@ -106,14 +104,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
 Microsoft.AspNetCore.StaticFiles,ship
 Microsoft.AspNetCore.TestHost,ship
 Microsoft.AspNetCore.Testing,noship
-Microsoft.AspNetCore.WebSockets.Client,ship
+Microsoft.AspNetCore.WebSockets.Client,noship
```

Comment:
cc @muratg  - this was part of https://github.com/aspnet/Coherence/commit/dd84f82db56ad63c3ff0b5e0122fe1ca250bcaaf that was unintentionally only checked in to dev.